include/dfrn.php

```diff
@@ -16,6 +26,10 @@
  */
 class dfrn {
 
+	const DFRN_TOP_LEVEL = 0;
+	const DFRN_REPLY = 1;
+	const DFRN_REPLY_RC = 2;
```

Comment:
You can use ```///< Comment``` at the and of the line to put documentation after members

Note:
> These blocks can only be used to document members and parameters. They cannot be used to document files, classes, unions, structs, groups, namespaces and enums themselves. Furthermore, the structural commands mentioned in the next section (like \class) are not allowed inside these comment blocks.

http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#memberdoc

That's why I couldn't do this for constants in boot.php :-/